go.mod

```diff
@@ -32,5 +35,25 @@ go 1.15
 
 replace github.com/lightninglabs/loop/swapserverrpc => ./swapserverrpc
 
-// TODO(bhandras): remove once v0.14.3-beta or v0.15.0-beta is tagged.
-replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.14.1-beta.0.20220131141659-60625b6c1a0b
+replace (
+	github.com/lightningnetwork/lnd => github.com/guggero/lnd v0.11.0-beta.rc4.0.20220309190327-b0f5361e48c1
+	github.com/lightningnetwork/lnd/cert => github.com/guggero/lnd/cert v1.0.4-0.20220309180237-7dfe4018ce2f
+	github.com/lightningnetwork/lnd/healthcheck => github.com/guggero/lnd/healthcheck v0.0.0-20220309180237-7dfe4018ce2f
+	github.com/lightningnetwork/lnd/tlv => github.com/guggero/lnd/tlv v0.0.0-20220309180237-7dfe4018ce2f
+)
+
+replace (
+	github.com/btcsuite/btcd => github.com/arshbot/btcd v0.22.0-beta.0.20220310092746-12c6b5570848
+	github.com/btcsuite/btcd/btcec/v2 => github.com/Roasbeef/btcd/btcec/v2 v2.0.0-20220225020451-4629cbd4eb52
+	github.com/btcsuite/btcd/btcutil => github.com/arshbot/btcd/btcutil v1.1.1-0.20220310092746-7d121ce37f2a
+	github.com/btcsuite/btcd/chaincfg/chainhash => github.com/arshbot/btcd/chaincfg/chainhash v0.0.0-20220310092746-97d068e90261
+)
+
+replace (
+	github.com/btcsuite/btcwallet => github.com/guggero/btcwallet v0.13.1-0.20220307185840-2b6e334b8c63
+	github.com/btcsuite/btcwallet/wallet/txauthor => github.com/guggero/btcwallet/wallet/txauthor v1.1.1-0.20220307185840-2b6e334b8c63
+)
+
+replace github.com/lightninglabs/lndclient => github.com/arshbot/lndclient v1.0.1-0.20220310212109-ce9e183be306
```

Comment:
From the build failure: 
```
routing_plugin.go:565:57: too many arguments (typecheck)

	err := r.lnd.Router.ImportMissionControl(ctx, entries, true)
```

I think you need to rebase your lndclient branch on `lndclient v0.14.2-3` so that it's got the new MC param?